Source/ZoomNet/Models/PaginatedResponseWithTokenAndDateRange.cs

```diff
@@ -0,0 +1,58 @@
+using Newtonsoft.Json;
+
+namespace ZoomNet.Models
+{
+	/// <summary>
+	/// Pagination Object.
+	/// </summary>
+	/// <typeparam name="T">The type of records.</typeparam>
+	public class PaginatedResponseWithTokenAndDateRange<T>
```

Comment:
This class can be greatly simplified by deriving from `PaginatedWithToken<T>` and simply adding the `From` and `To` properties

```suggestion
	public class PaginatedResponseWithTokenAndDateRange<T> : PaginatedResponseWithToken<T>
```